docs/versioning.rst

```diff
@@ -15,16 +15,41 @@ Release support
    * - Release
      - :ref:`Support level<versioning_support_levels>`
      - Minimum Datadog Agent
-   * - 0.x
+   * - ``<1``
      - :ref:`Maintenance<versioning_support_maintenace>`
      -
-   * - 1.x
+   * - ``>=1.0,<2``
      - :ref:`General Availability<versioning_support_ga>`
      - 7.28
 
 
-Versions
-========
+.. _versioning_support_levels:
+
+Support levels
+==============
+
+
+.. list-table::
+   :header-rows: 1
+
+   * - Level
+     - Policy
+
+       .. _versioning_support_ga:
+   * - General Availability (GA)
+     - Full implementation of all features. Full support for new features, bug and security fixes.
```

Comment:
I'm not 100% sure if "full" is well defined here or what "Full implementation of all features" means